viash/core/src/project/create_component/script.py

```diff
@@ -62,59 +61,77 @@ def create_config(par, component_type, pretty_name, script_path) -> str:
     |'''
   )
 
-def generate_info(par, component_type, pretty_name) -> str:
-  """Generate the info for a component."""
+def generate_general_info(par, component_type, pretty_name) -> str:
+  """Generate the general info for a method."""
+  str = strip_margin(f'''\
+    |# A unique identifier for your component (required).
+    |# Can contain only lowercase letters or underscores.
+    |name: {par["name"]}
+    |''')
   if component_type in ["method", "control_method"]:
-    str = strip_margin(f'''\
-      |  # A relatively short label, used when rendering visualisations (required)
-      |  label: {pretty_name}
-      |  # A one sentence summary of how this method works (required). Used when 
-      |  # rendering summary tables.
-      |  summary: "FILL IN: A one sentence summary of this method."
-      |  # A multi-line description of how this component works (required). Used
-      |  # when rendering reference documentation.
-      |  description: |
-      |    FILL IN: A (multi-line) description of how this method works.
-      |  # Which normalisation method this component prefers to use (required).
-      |  preferred_normalization: log_cp10k
+    str += strip_margin(f'''\
+      |# A relatively short label, used when rendering visualisations (required)
+      |label: {pretty_name}
+      |# A one sentence summary of how this method works (required). Used when 
+      |# rendering summary tables.
+      |summary: "FILL IN: A one sentence summary of this method."
+      |# A multi-line description of how this component works (required). Used
+      |# when rendering reference documentation.
+      |description: |
+      |  FILL IN: A (multi-line) description of how this method works.
       |''')
     if component_type == "method":
       str += strip_margin(f'''\
-        |  # A reference key from the bibtex library at src/common/library.bib (required).
-        |  reference: bibtex_reference_key
+        |# A reference key from the bibtex library at src/common/library.bib (required).
+        |references:
+        |  doi: doi_reference_key
+        |  # Can also be a bibtex: see https://viash.io/versioned/0_9_0/reference/config/references.html#bibtex                 
```

Comment:
I think it's better to give an example in comments. Also the `library.bib` is now not used anymore :D

```suggestion
        |# references:
        |#   doi: 
        |#     - 10.1000/xx.123456.789
        |#   bibtex:
        |#     - |
        |#       @article{foo,
        |#         title={Foo},
        |#         author={Bar},
        |#         journal={Baz},
        |#         year={2024}
        |#       }
```

Reply:
I was thinking of doing the same but are the `{}` not going to give errors ?

Reply:
good point! Fixed!